appveyor.yml

```diff
@@ -6,7 +6,7 @@ environment:
 
 install:
   - ps: Install-Product node $env:nodejs_version
-  - npm -g install npm@3 && set PATH=%APPDATA%\npm;%PATH%
+  - npm -g install npm@6 && set PATH=%APPDATA%\npm;%PATH%
   - npm install
```

Comment:
Can we change this to `npm ci`? That way it will fail the build if the `package-lock.json` file isn't up to date with the `package.json` file

Reply:
Good idea 👍 just committed this change for both travis and appveyor